go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/tendermint/tendermint => github.com/pokt-network/tendermint v0.32.11-0.20220713165017-a165a87b48af
+//replace github.com/tendermint/tendermint => github.com/pokt-network/tendermint v0.32.11-0.20220420160934-de1729fc7dba
+replace github.com/tendermint/tendermint => ../tendermint
```

Comment:
By using the project fails to build by running `go build`
`replace github.com/tendermint/tendermint => ../tendermint` should be replaced by latest tendermint modification
`replace github.com/tendermint/tendermint => github.com/fiagdao/tendermint@v0.32.11-0.20220822131801-2169e8766dac` but that too seems to be unbuildable due to problems on the go mod. Please check